website/www/site/assets/scss/_hero.scss

```diff
@@ -24,7 +24,8 @@
    width: 100%;
    height: 100%;
    display: inherit;
-   margin-top: -30px;
+   //usually -30px, change for 0 until summit is over and banner is removed
```

Comment:
Let's format this comment as a todo to make it easier to find later:

`// TODO: revert to -30px after summit is over`